pkg/update/release_image.go

```diff
@@ -196,6 +196,9 @@ func (s ReleaseImageSpec) calculateImageUpdates(rc ReleaseContext, candidates []
 		ref, err = s.ImageSpec.AsRef()
 		if err == nil {
 			singleRepo = ref.CanonicalName()
+			// FIXME(fons): we probably want to allow this operation even if image
+			//              scanning is disabled. We could either avoid the validation
+			//              or use an uncached registry.
```

Comment:
The uncached registry is not an option, Michael shared insights about this in another issue (Flux had this in the beginning but it lead to timeouts).

Reply:
That code only validates that the images exist, I think we can use an uncached registry for that. Otherwise releasing a specific image won't work.